### PR TITLE
Restructure Docs

### PR DESCRIPTION
**Pull request by svollath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,10 +59,47 @@ to use Kubernetes as a PaaS (Platform as a Service).
...
+
+Refer to [Install the Epinio CLI](./docs/user/tutorials/install_epinio_cli.md).
+
+### Installation Methods (in Cluster)
+
+Beside advanced installation options, there are two ways of installing Epinio:
+
+1. [Installation using a MagicDNS Service](./docs/user/tutorials/install_epinio_magicDNS.md)
+
+- For test environments. This should work on nearly any kubernetes distribution. Epinio will try to automatically create a magic DNS domain, e.g. **10.0.0.1.omg.howdio.website**.
```
Comment (andreas-kupries) [inline README.md:91]:
```suggestion
- For test environments. This should work on nearly any kubernetes distribution. Epinio will try to automatically create a magic DNS domain, e.g. **10.0.0.1.omg.howdoi.website**.
```

File: README.md [hunk 2]
```diff
@@ -59,10 +59,47 @@ to use Kubernetes as a PaaS (Platform as a Service).
...
+
+2. [Installation using a Custom Domain](./docs/user/tutorials/install_epinio_customDNS.md)
+
+- For test and production environments. You will define a system domain, e.g. **test.example.com**.
```
Comment (andreas-kupries) [inline README.md:95]:
```suggestion
- For test and production environments. You will have to define a system domain, e.g. **test.example.com**.
```

File: docs/user/tutorials/install_epinio_cli.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+##### MacOS
+
+```bash
+curl -o epinio -L https://github.com/epinio/epinio/releases/download/<epinio-version>/epinio-darwin-amd64
+```
+
+##### Windows
+
+```bash
+ curl -LO https://github.com/epinio/epinio/releases/download/<epinio-version>/epinio-windows-amd64.exe
```
Comment (andreas-kupries) [inline docs/user/tutorials/install_epinio_cli.md:31]: This is somewhat different in the options from the `curl` commands for linux and macos above.
Are these differences significant ?
If not why are they present ?
If yes, would the user need an explanation of why ?

**Review by andreas-kupries — CHANGES_REQUESTED** [review]

File: docs/user/tutorials/install_epinio_cli.md (hunk 3 above)
Comment (svollath) [reply docs/user/tutorials/install_epinio_cli.md:31]: This part is copied from the existing documentation - it's identical to the current documentation, so no change. Assuming it was tested on Windows, and documented accordingly.

**Review by manno — APPROVED** [review]

**Pull request merged**